http.go

```diff
@@ -0,0 +1,52 @@
+package authcontrol
+
+import (
+	"maps"
+	"net/http"
+	"time"
+
+	"github.com/go-chi/jwtauth/v5"
+	"github.com/go-chi/traceid"
+	"github.com/go-chi/transport"
+	"github.com/lestrrat-go/jwx/v2/jwt"
+)
+
+// Service-to-service HTTP client for internal communication between Sequence services.
+func S2SClient(serviceName string, jwtSecret string, debugRequests bool) *http.Client {
+	httpClient := &http.Client{
+		Transport: transport.Chain(http.DefaultTransport,
+			traceid.Transport,
+			transport.SetHeaderFunc("Authorization", s2sAuthHeader(jwtSecret, map[string]any{"service": serviceName})),
+			transport.If(debugRequests, transport.LogRequests(transport.LogOptions{Concise: true, CURL: true})),
+		),
+	}
+
+	return httpClient
+}
+
+// Create short-lived service-to-service JWT token for internal communication between Sequence services.
+func S2SToken(jwtSecret string, claims map[string]any) string {
```

Comment:
Here I'm thinking if we should define a struct for the claims.. rather than generic `map[string]any`. It'd be nice to json marshal/unmarshal a known type across our services.. but I'm open to suggestions. Thoughts?

Reply:
That's a really good idea.
Maybe we can do that in a separate PR first

Reply:
OK, let's do this in follow-up PR, @david-littlefarmer 

For now, I think we just need "service" claim and "accountId" (aka walletAddr? -- check in stack/guard).